filclient/filclient.go

```diff
@@ -760,7 +761,7 @@ func (fc *FilClient) CheckOngoingTransfer(ctx context.Context, miner address.Add
 
 }
 
-func (fc *FilClient) RetrievalQuery(ctx context.Context, maddr address.Address, pcid cid.Cid) (*retrievalmarket.QueryResponse, error) {
+func (fc *FilClient) RetrievalQuery(ctx context.Context, maddr address.Address, pcid cid.Cid, optionalSelector ipld.Node) (*retrievalmarket.QueryResponse, error) {
```

Comment:
is this even used?

Reply:
Not at present: https://github.com/filecoin-project/go-fil-markets/blob/32e5ccef50d88d8a19b79f71ea98157c4a0047d2/retrievalmarket/types.go#L163

But in the future they very well might be able to respond intelligently, so I wired it through into the signature.